src/app/app.service.ts

```diff
@@ -67,20 +70,20 @@ export class AppService {
      */
     public static async initAgent(app: INestApplication) {
         const agentService = await app.resolve(AgentService);
-        agentService.initProfilesFromDisk();
+        await agentService.initProfilesFromDisk();
```

Comment:
This change concerns me because its been without the await operator for a long time.  maybe this was intentional to let it go run without waiting?

Reply:
Could be. There are enough `require-await` linting errors in our codebase that we might want to think about just turning it off by default across our apps.